auv_smach/scripts/main_state_machine.py

```diff
@@ -42,6 +44,12 @@ def __init__(self):
 
         rospy.Subscriber("/taluy/propulsion_board/status", Bool, self.enabled_callback)
 
+        self.cancel_align_controller_service = rospy.ServiceProxy(
+            "/taluy/control/align_frame/cancel", Trigger
+        )
+
+        signal.signal(signal.SIGINT, self.shutdown_handler)
```

Comment:
I believe the SIGINT signal already puts the main state machine in to preempted outcome, maybe when that outcome is achieved we just call the align frame controller and cancel it? Just asking if this signal handling is necessary

Reply:
SIGINT puts the main state machine to aborted outcome.

I think we can set each state's aborted state to  `CancelAlignControllerState` we already have. I think this may be a better way than signal handling.